awx/ui_next/src/screens/Template/JobTemplateDetail/JobTemplateDetail.jsx

```diff
@@ -168,11 +176,11 @@ class JobTemplateDetail extends Component {
             <Detail label={i18n._(t`Timeout`)} value={timeout || '0'} />
             <Detail
               label={i18n._(t`Created`)}
-              value={`${created} by ${summary_fields.created_by.username}`} // TODO: link to user in users
+              value={`${modified} ${createdBy}`} // TODO: link to user in users
```

Comment:
This is going to be nit-picky so forgive me but this approach is a bit limiting from a translations perspective.  Ideally we'd keep the entire string together `${createdTimestamp} by ${username}` to let the translation team determine the ordering of the pieces.  We might have to something like:

```
const modifiedBy = summary_fields.modified_by && summary_fields.modified_by.username ? modified : i18n._(t`${modified} by ${summary_fields.modified_by.username}`);
```

up above in order to achieve this. 

I also think `${modified} ${createdBy}` should be `${created} ${createdBy}`.  It seems like the modified timestamp is shown with created by and created timestamp shown with modified by.

Reply:
:+1: sounds good to me and good catch. Should be fixed now.